qr-code-auth/README.md

```diff
@@ -0,0 +1,11 @@
+# QrCodeAuth
+
+```
+{
+  'user_id': string,
+  'device_info':string,
+  'is_authorized': boolean,
```

Comment:
What does this specifies?

Reply:
Whether the user has authorized the mobile device to be signed in

Reply:
yup